dataactbroker/handlers/userHandler.py

```diff
@@ -58,12 +58,12 @@ def deleteToken(self,token):
         self.session.delete(oldToken)
         self.session.commit()
 
-    def getUsers(self, agency=None):
+    def getUsers(self, cgac_code=None):
```

Comment:
Same question throughout the userHandler functions re: surrogate key vs using the CGAC code